examples/src/SelectorDropdownExamples.js

```diff
@@ -132,6 +132,36 @@ const SelectorDropdownExamples = () => (
             'Straight Outta Compton',
             'Reasonable Doubt',
             'Super long name that should be truncated, we should see the dots at the very end, adding some more text here, should truncate at any second now, please truncate soon',
+            'Illmatic',
```

Comment:
Just increasing the size of the dropdown, can remove or edit it to make more sense if needed